### ISSUE TITLE
Can't remove members when group size < 3

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#944]
Note: I found this issue while reviewing PR #916 but it also happens on master.

### Problem

Here's what I did to find this bug:
1. Create group;
2. Add 2 more members to it (proposals are now enabled);
3. Remove 1 member (via proposal that was accepted with 1 yes vote);
4. Try to remove 1 more member, so my group can have only 1 member, created a proposal;
5. Impossible to close that proposal because the proposal creator can’t vote and the member being removed can’t vote either (and the group only has 2 members);

### Solution

This is what I expected to happen:

- Remove first member via proposal, because the group has 3 members;
- Remove the second member immediately, without proposal.

Let me know if you agree with my suggestion!

**Comment by dotmacro** [comment]
>  Remove first member via proposal, because the group has 3 members;  

Yes, exactly.

>  Remove the second member immediately, without proposal.  
No, because that's merely a race to see who's left. If there are two remaining members, whoever thinks to kick the other out first "wins". 

In a group of two, it makes more sense that anyone who is unhappy is free to either stay and try to improve the group (e.g. inviting new members, working things out with the other member), or voluntarily leave the group and start a new group. 

> 4. Try to remove 1 more member, so my group can have only 1 member, created a proposal;

The bug is that it's possible to create such a proposal when there are only 2 group members :)

**Comment by sandrina-p** [comment]
> Try to remove 1 more member, so my group can have only 1 member, created a proposal;

How did you create a proposal if the group has only 2 members? It should be immediately ~~(and it is).~~ Wow, yeah, this is a bug, since #900. It is immediate by default (correct), but if a member left before it's still a proposal (bug). This happens with any kind of proposal by the way.

When the "Remove Member" feature was implemented (#823), @taoeffect suggested that in a group of 2, only the creator can remove the other member and that's how it was implemented back then.

While playing around, I found another bug (related to #850). If we create any proposal with >=3 members and then the members are no longer part of the group, it's impossible to close the proposal. That's a funny edge case. We still need to add some side-effects to handle these scenarios, something like:
- All proposals should be closed/canceled once the group goes back to 2 members again.
- Open proposals created by a removed member should be canceled automatically.
